src/Microsoft.OpenApi.OData.Reader/Edm/ODataPathProvider.cs

```diff
@@ -366,9 +366,9 @@ private void RetrieveNavigationPropertyPaths(IEdmNavigationProperty navigationPr
                 // ~/entityset/{key}/single-valued-Nav/subtype
                 CreateTypeCastPaths(currentPath, convertSettings, navigationProperty.DeclaringType, navigationProperty, targetsMany);
 
-                if (!navigationProperty.ContainsTarget)
+                if (navigation?.Referenceable == true)
```

Comment:
this translates to requiring explicit annotations.
i.e. if no annotation value or if an annotation value is present and false, don't add the path items,
if if an annotation value is present and true, add the path items.
Which will require workloads to explicitly tell us the ref endpoints are supported.
I personally don't mind but wanted to check first with @mikepizzo and @darrelmiller whether this was the desired behavior?

Reply:
I would prefer us to be opt-in.  I would rather add the paths that we know are implemented than remove those that are not.  It's not the original OData philosophy but it is more natural for OpenAPI generation.

Reply:
@darrelmiller what does your suggestion mean in terms of implementation? Do we check explicitly for `Referenceable=true` annotations before creating `$ref` paths and nullability or `Referenceable=false` means no `$ref` paths to be created?

Reply:
$ref paths should only be created if Referenceable is present and true.  I don't know why nullability is relevant.

Reply:
thanks for clarifying the intent here, no changes required, resolving